specification/blueprint/resource-manager/Microsoft.Blueprint/preview/2018-11-01-preview/assignmentOperation.json

```diff
@@ -0,0 +1,322 @@
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/ScopeParameter"
+          },
+          {
+            "$ref": "#/parameters/AssignmentNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK - all Blueprint assignment Operation retrieved.",
+            "schema": {
+              "$ref": "#/definitions/AssignmentOperationList"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/{scope}/providers/Microsoft.Blueprint/blueprintAssignments/{assignmentName}/operations/{assignmentOperationName}": {
+      "get": {
+        "tags": [
+          "AssignmentOperations"
+        ],
+        "operationId": "AssignmentOperations_Get",
+        "description": "Get a Blueprint assignment operation.",
+        "x-ms-examples": {
+          "Assignment": {
+            "$ref": "./examples/BlueprintAssignmentOperation_Get.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/ScopeParameter"
+          },
+          {
+            "$ref": "#/parameters/AssignmentNameParameter"
+          },
+          {
+            "$ref": "#/parameters/AssignmentOperationNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK - Blueprint assignment retrieved.",
+            "schema": {
+              "$ref": "#/definitions/AssignmentOperation"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "AssignmentOperation": {
+      "type": "object",
+      "description": "Represents underlying deployment detail for each update to the assignment.",
+      "allOf": [
+        { "$ref": "#/definitions/AzureResourceBase" }
+      ],
+      "properties": {
+        "properties": {
+          "description": "Properties for AssignmentOperation",
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/AssignmentOperationProperties"
+        }
+      }
+    },
+    "AssignmentOperationList":{
+      "type": "object",
+      "description": "List of AssignmentOperation.",
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "List of AssignmentOperation.",
+          "items": {
+            "$ref": "#/definitions/AssignmentOperation"
+          }
+        },
+        "nextLink": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Link to the next page of results."
+        }
+      }
+
+    },
+    "AssignmentOperationProperties": {
+      "type": "object",
+      "description": "properties of AssignmentOperation.",
+      "properties": {
```

Comment:
Are any of these properties readonly? 
If they are computed in the RP, mark them as readonly so the CLI doesn't require them as input.

Reply:
this is Get/List only, I don't have the readonly problem.